shared/reducers/login.js

```diff
@@ -73,7 +74,8 @@ const initialState: LoginState = {
     onSubmit: () => {},
     existingDevices: [],
     deviceName: ''
-  }
+  },
+  configuredAccounts: []
```

Comment:
Shouldn't this be null?

With null meaning you haven't fetched the data vs an empty array meaning the data has been fetched and is empty